### ISSUE TITLE
Command Name Discussion

### ISSUE THREAD

**Issue by amayer5125** [cakephp/queue#73]
Looking over this plugin I'm wondering if the queue commands should be namespaced. Something like `bin/cake queue work` or `bin/cake queue worker` may allow for future improvements if we wanted to add something like `bin/cake queue flush <queue_name>`.

This would also have the benefit of being easier to identify when looking at processes on a system. As a system administrator, seeing `bin/cake worker` implies that it is a cake process, but is not helpful in relaying what is being worked. Is it some kind of file watcher worker or a PDF generator worker? Seeing `bin/cake queue work` would give clarity in a system process list.

**Comment by markstory** [comment]
Sounds like a good idea to me. We should also retain the old aliases for a while to allow users to upgrade.

**Comment by amayer5125** [comment]
I would be happy to make a pull request, but I can't seem to find any documentation on how to make command aliases. Is there an example or specific documentation you could point me to for how you would like this implemented?

**Comment by ADmad** [comment]
@amayer5125 You need to essentially duplicate this line and use `'queue worker'` instead of `'worker'` https://github.com/cakephp/queue/blob/5204caba481679864e82c2a99f19f8f90d83e253/src/Plugin.php#L67

**Comment by amayer5125** [comment]
I gave that a try but it didn't display `queue work` as an option when running `bin/cake --help`. The output always showed `worker` as the only option.

I tried both of these variations, but none showed `queue work` as the default.

```php
return $commands
    ->add('queue work', WorkerCommand::class)
    ->add('worker', WorkerCommand::class);

// and

return $commands
    ->add('worker', WorkerCommand::class)
    ->add('queue work', WorkerCommand::class);
```

Running `bin/cake --help | grep -A 2 -F Cake/Queue` always output:
```
Cake/Queue:
 - worker
(blank line here)
```

I did confirm that I could run `bin/cake queue work` even though it wasn't listed in the `--help` options. Do we care if it is listed as the default when running `bin/cake --help` or is updating the README good enough?
